### DISCUSSION TITLE
Transient became part of which Emacs release?

### DISCUSSION THREAD

**Discussion by doolio** [magit/transient#263] [Support] [answered]
The `transient.el` header states "This file is part of GNU Emacs." but which Emacs release was this? I ask as I am trying to extend another package and wish to use `transient` as a dependency. This other packages supports at a minimum Emacs 26.1 and so I think my extension should do the same. With that do I need to list `transient` in my "Package Requires:" line? 

Thank you for `transient` and your time.

P.S. I did quickly scan the release notes here but didn't see it explicitly mentioned.

**Comment by tarsius** [comment, marked as answer]
It was added in 28.1, but you should just add `(transient "0.4.3")` to `Package-Requires` anyway, to ensure the latest version is installed (which after this weekend will be `0.5.0`).

**Reply by doolio** [reply]
Perfect. Thanks Jonas and for the quick response.
